packages/isomorphic/README.md

```diff
@@ -0,0 +1,102 @@
+# @xrplf/isomorphic
+
+A collection of isomorphic implementations of crypto and utility functions.
+
+Browser implementations of cryptographic functions use `@noble/hashes` and `crypto` for node .
+
+### Hashes
+
+All hash functions (similar to `@noble/hashes`):
+
+- can be called directly, with Uint8Array.
+- return Uint8Array
+- can receive string, which is automatically converted to Uint8Array via utf8 encoding (not hex)
+
+```
+function hash(message: Uint8Array | string): Uint8Array;
+hash(new Uint8Array([1, 3]));
+hash('string') == hash(new TextEncoder().encode('string'));
+```
+
+All hash functions can be constructed via `hash.create()` method:
+
+- the result is `Hash` subclass instance, which has `update()` and `digest()` methods
+- `digest()` finalizes the hash and makes it no longer usable
+
+```typescript
+hash
+  .create()
+  .update(new Uint8Array([1, 3]))
+  .digest();
+```
+
+### `@xrplf/isomorphic/ripemd160`
+```typescript
+import { ripemd160 } from '@xrplf/isomorphic/ripemd160';
+const hash8 = ripemd160('abc');
+const hash9 = ripemd160()
+  .create()
+  .update(Uint8Array.from([1, 2, 3]))
+  .digest();
+```
+
+
+### `@xrplf/isomorphic/sha256`
+
+```typescript
+import { sha256 } from '@xrplf/isomorphic/sha256';
+const h1a = sha256('abc');
+const h1b = sha256
+  .create()
+  .update(Uint8Array.from([1, 2, 3]))
+  .digest();
+```
+
+### `@xrplf/isomorphic/sha512`
+
+```typescript
+import { sha512 } from '@xrplf/isomorphic/sha512';
+const h2a = sha512('abc');
+const h2b = sha512
+  .create()
+  .update(Uint8Array.from([1, 2, 3]))
+  .digest();
+```
+
+## Utilities
+
+### `@xrplf/isomorphic/utils`
+
+#### randomBytes
+
+Create an UInt8Array of the supplied size
+
+```typescript
+import { randomBytes } from @xrplf/isomorphic/utils
+
+console.log(randomBytes(12)) // Uint8Array(12) [95, 236, 188,  55, 208, 128, 161, 249, 171, 57, 141, 7]
+```
+
+#### bytesToHex
+
+Convert an UInt8Array to hex.
+
+```typescript
+import { bytesToHex } from @xrplf/isomorphic/utils
+
+console.log(bytesToHex([222, 173, 190, 239])) // "DEADBEEF"
+```
+
+#### hexToBytes
+
+Convert hex to an UInt8Array.
+
+```typescript
+import { hexToBytes } from @xrplf/isomorphic/utils
+
+console.log(hexToBytes('DEADBEEF')) // [222, 173, 190, 239]
+```
+
+### `@xrplf/isomorphic/ws`
+
+// TODO: Websocket Wrapper and `ws`
```

Comment:
Bump - Should this be resolved before the PR is merged?

Reply:
I have a branch ready for this change once this PR gets merged.  It does not need to be resolved until later.

Reply:
https://github.com/XRPLF/xrpl.js/tree/feature/ws-iso-2